ts/features/fims/history/components/Loaders.tsx

```diff
@@ -0,0 +1,25 @@
+import { Divider } from "@pagopa/io-app-design-system";
+import * as React from "react";
+import { View } from "react-native";
+import Placeholder from "rn-placeholder";
+
+export const LoadingFimsHistoryItemsFooter = ({
```

Comment:
There is no need for such complexity. Just link the `LoadingFimsHistoryListItem` once. You can also omit the `Divider`

Reply:
By design, 3 loading skeletons are requested.
That said, there is still very much space for debate, and we will have many opportunities to bring this up in a team sync.